installer/install.sh

```diff
@@ -83,15 +83,33 @@ fi
 # get running machine from conf file
 if [ -r /etc/machine.conf ]; then
     read_conf_file "/etc/machine.conf"
+    machine_conf_path=/etc/machine.conf
 elif [ -r /host/machine.conf ]; then
     read_conf_file "/host/machine.conf"
+    machine_conf_path=/host/machine.conf
 elif [ "$install_env" != "build" ]; then
     echo "cannot find machine.conf"
     exit 1
 fi
 
+onie_running_arch=`cat $machine_conf_path | grep onie_arch | sed -n "s/^onie_arch=\(.*\)$/\1/p"`
```

Comment:
>onie_running_arch [](http://example.com/codeflow?start=0&length=17)

`read_conf_file` function is a general parsing function for `etc/machine.conf`. So you do not need to parse these variables manually.